test/tests/misc/solution_invalid/tests

```diff
@@ -29,10 +29,16 @@
       detail = 'unless the user specifies to silence them,'
     []
     [console_output]
-      type = 'JSONDiff'
+      type = 'RunApp'
       input = 'solution_invalid.i'
       cli_args = "Materials/filter/flag_solution_warning=true Outputs/file_base='solution_invalid_warning'"
-      jsondiff = 'solution_invalid_warning.json'
```

Comment:
any reason for the change?
you can have an expect_out on a JSONDiff

Reply:
My changes got loss using `--reapply-cherry-picks`. Will fix it